demo/pages/_app.js

```diff
@@ -1,10 +1,32 @@
 /* eslint-disable react/prop-types */
-import '@moxy/react-lib-template/dist/index.css';
+import React, { useEffect, useCallback, useRef } from 'react';
+import PageSwapper from '@moxy/react-page-swapper';
+import { getScrollBehavior } from '@moxy/next-scroll-behavior';
+import PageTransition from '../shared/modules/react-page-transition';
 
-import React from 'react';
+import styles from './_app.module.css';
 
-const App = ({ Component, pageProps }) => (
-    <Component { ...pageProps } />
-);
+if (typeof history !== 'undefined') {
+    history.scrollRestoration = 'manual';
+}
+
+const App = ({ Component, pageProps }) => {
+    const ScrollBehaviorRef = useRef();
+
+    useEffect(() => {
+        ScrollBehaviorRef.current = getScrollBehavior();
```

Comment:
I think you should return `() => scrollBehaviorRef.stop()` in the `useEffect` so that it's properly cleaned.